conversion.py

```diff
@@ -5,4 +5,7 @@ def celsius_to_kelvin(celsius):
     return celsius + 273.15
 
 def kelvin_to_celsius(kelvin):
-    return kelvin - 273.15
\ No newline at end of file
+    return kelvin - 273.15
+
+def celsius_to_fahrenheit(celsius):
+    return celsius * 9./5 + 30
```

Comment:
```suggestion
    return celsius * 9./5 + 32
```

This should be 32! :D